packages/presentation/lib/screen/home/widgets/home_grid_view.dart

```diff
@@ -69,22 +73,21 @@ class _HomeGridViewState extends State<HomeGridView> {
                       crossAxisAlignment: CrossAxisAlignment.start,
                       children: [
                         MovieImage(image: currentMovie.images),
-                        const Spacer(),
+                        SizedBox(height: Dimens.size10.h),
```

Comment:
better .h .w move to Class Dimens, you cannot create const variable but you can use final

Reply:
Why? I guess better to use .h and .w and don't make new variables in Dimens class

Reply:
Alex is right. it would allow u to avoid writing .h/.w each time